restservice/src/main/java/org/itevents/service/security/AuthenticatedUser.java

```diff
@@ -0,0 +1,57 @@
+package org.itevents.service.security;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+
+/**
+ * Created by ramax on 1/28/16.
+ */
+public class AuthenticatedUser implements UserDetails {
+
+    private Collection<? extends GrantedAuthority> authorities;
+    private String token;
+    private String username;
+
+    public AuthenticatedUser(String username, String token, Collection<? extends GrantedAuthority> authorities) {
+        this.username = username;
+        this.token = token;
+        this.authorities = authorities;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return authorities;
+    }
+
+    @Override
+    public String getPassword() {
+        return token;
```

Comment:
Field and getter have different names. Maybe it is better to reneme setter or field?